README.md

```diff
@@ -2,41 +2,37 @@
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
-An api that returns double-dummy results for a given board.
+An api that returns double-dummy results for a given deal.
 
 ## Usage
 ```
 curl --header "Content-Type: application/json" --request POST --data '{"hands":{"S":["D3", "C6", "DT", "D8", "DJ", "D6", "CA", "C3", "S2", "C2", "C4", "S9", "S7"],"W":["DA", "S4", "HT", "C5", "D4", "D7", "S6", "S3", "DK", "CT", "D2", "SK","H8"],"N":["C7", "H6", "H7", "H9", "CJ", "SA", "S8", "SQ", "D5", "S5", "HK", "C8", "HA"],"E":["H2", "H5", "CQ", "D9", "H4", "ST", "HQ", "SJ", "HJ", "DQ", "H3", "C9", "CK"]}}' https://dds.prod.globalbridge.app/api/dds-table/
 ```
-Uses Bo Hagland's solver https://github.com/dds-bridge/dds -- requires the libdds.so (or dds.dll in windows) to be installed and accessible.
-Credit to Alexis Rimbaud of NukkAI for the python dds wrapper.
-
-
-## Build the C++ library for local testing
-
+or equivalently
 ```
-make libdds-build
+make curl_prod
 ```
+Uses Bo Hagland's solver https://github.com/dds-bridge/dds -- requires libdds.so (dds.dll on MS-Windows, libdds.2.dylib on MacOS.) The *make* targets `build` and `start_local_server` will build this library for you. 
 
-The python loader looks for the library from libdds/.build/src. If the file is
-found from build directory then the found library is preferred before a library
-in a system directory.
-
-If you want to change the C++ library's build configuration,
-[README.libdds.md](README.libdds.md) provides information about the process.
+Our thanks to Alexis Rimbaud of NukkAI for the Python dds wrapper.
 
 ## Install a local server using Flask, then test it manually:
 
 ```
 pip3 install -r requirements.txt
-python3 -m src.api
+
+make start_local_server
+
 # In a separate terminal window…
 make curl_local
 ```
 
+If you want to change the C++ library's build configuration,
+[README.libdds.md](README.libdds.md) provides information about the process.
+
 ### Configuration ###
 
-The server supports configuration using `server.yaml` file. Configuration file is
+The server supports optional configuration using a `server.yaml` file. The onfiguration file is
```

Comment:
s/onfiguration/configuration/

Reply:
Though I'd fixed this! Maybe I didn't push my last change.

Reply:
Fixed.